posthog/queries/test/__snapshots__/test_trends.ambr

```diff
@@ -4364,6 +4371,13 @@
           (SELECT count() AS intermediate_count,
                   toStartOfDay(toTimeZone(toDateTime(timestamp, 'UTC'), 'UTC')) AS date
            FROM events e
+           INNER JOIN
```

Comment:
hmmmm, this is annoying, the non-poe version gets pdi joins, despite just grouping by distinct id.

@yakkomajuri were we supposed to group by person id here in poe-v2? 🤔

Reply:
actually, yes. I think the existing test is borked, where it's grouped by distinct id. Should be person id

Reply:
Nevermind, I'm getting confused, this is the group case, not person case, so needs fixing 🤦